### PR TITLE
feat: new editors in popup. TNL-10743

### PR DESCRIPTION
**Pull request by kenclary**
feat: new editors in popup. TNL-10743

### CODE REVIEW

**Review by connorhaugh — COMMENTED** [review]
Looks good to merge if unblocking ray is paramount, but that doesn't seem to be the case atm, so I would like to review this when it is passing tests. If @rayzhou-bit  is blocked he may want to review

**Comment by rayzhou-bit** [comment]
I'm getting an error with i18n when trying to run this.
image

File: src/library-authoring/author-library/LibraryAuthoringPage.jsx [hunk 1]
```diff
@@ -89,9 +89,8 @@ export const BlockPreviewBase = ({
...
         <ActionRow>
           <Button
-            as={Link}
-            to={editView}
-            variant="tertiary"
+            aria-label={intl.formatMessage(messages['library.detail.block.edit'])}
+            onClick={() => setShowEditorModal(true)}
           >
             <FontAwesomeIcon icon={faEdit} className="pr-1" />
             {intl.formatMessage(messages['library.detail.block.edit'])}
```
Comment (rayzhou-bit) [inline src/library-authoring/author-library/LibraryAuthoringPage.jsx:96]: Should this use a FormattedMessage component?

**Comment by rayzhou-bit** [comment]
> I'm getting an error with i18n when trying to run this. 

Sorry, ignore this. This is due to my local devstack not working again.

File: src/library-authoring/author-library/LibraryAuthoringPage.jsx (hunk 1 above)
Comment (kenclary) [reply src/library-authoring/author-library/LibraryAuthoringPage.jsx:96]: it's only formatting (internationalizing) the aria-label. The button itself is just an icon. (Also, we're not trying to refactor all the existing UI/UX, just replacing the older LA editor pages with an FLCC editor popup)

**Comment by kenclary** [comment]
codecov has been fixed.

**Review by connorhaugh — APPROVED** [review]

**Pull request merged**